src/functionaltest/java/com/ericsson/ei/frontend/pageobjects/TestRulesPage.java

```diff
@@ -0,0 +1,24 @@
+package com.ericsson.ei.frontend.pageobjects;
+
+import java.io.IOException;
+
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class TestRulesPage extends PageBaseClass {
+
+    public TestRulesPage(CloseableHttpClient mockedHttpClient,
+            WebDriver driver, String baseUrl) throws ClientProtocolException, IOException {
+        super(mockedHttpClient, driver, baseUrl);
+    }
+
+    public String getMainHeader() {
+        WebElement mainHeader = driver.findElement(By.xpath("/html/body/div[3]/div[1]/div[2]/div[1]/h1"));
```

Comment:
How does this work? If I add a new div tag in the middle of this to html code,, will that break the tests?
Is it possible to connect this to an id parameter in the div tag instead.

Reply:
That is true. The specific element doesn't have ID or class identifier though, i guess we can add one instead of using xpath.